### PR TITLE
fix szokodiakos/typegoose#193

### PR DESCRIPTION
**Pull request by bangbang93**
fix fix szokodiakos/typegoose#193

### CODE REVIEW

**Comment by aljazerzen** [comment]
Nice job :+1: 

@Ben305 this looks good. I think it can be merged (I also need this funcionality)

File: src/test/models/user.ts [hunk 1]
```diff
@@ -74,7 +74,7 @@ export class User extends Typegoose {
   @arrayProp({ items: String, required: true })
   languages: string[];
 
-  @arrayProp({ items: Job })
+  @arrayProp({ items: Job, required:true })
   previousJobs?: Job[];
```
Comment (Ben305) [inline src/test/models/user.ts]: An optional array with a required decorator? Does that make sense?

**Comment by hasezoey** [comment]
this can be closed, because  #302 already included all the changes
@Ben305

**Comment by hasezoey** [comment]
@Ben305 [Bump](https://github.com/szokodiakos/typegoose/pull/194#issuecomment-514951471)

**Pull request closed without merging**